.github/workflows/test.yml

```diff
@@ -80,3 +80,37 @@ jobs:
 #      - 
 #        name: List bundle contents
 #        run: ls -R bundle/
+
+  test_template:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+
+      # Install saucectl
+      - uses: saucelabs/saucectl-run-action@v1
+        with:
+          skip-run: true
+
+      - name: Install yq
+        run: sudo snap install yq
+
+      # There is discrepencies between uid used to clone and uid in container,
+      # As a temporary measure, we chmod files to allow test outputs to be written.
+      - name: Workaround for container permissions
+        run: sudo chmod -R 777 tests/
+
+      # Run test in template
+      - run: |
+          docker build -t saucelabs/stt-testcafe-node:local . > /dev/null 2>&1
+          cd .saucetpl
+          yq e -i '.docker.image = "saucelabs/stt-testcafe-node:local"' .sauce/config.yml
+          yq e -i '.docker.fileTransfer = "copy"' .sauce/config.yml
+          yq e -i '.testcafe.version = "latest"' .sauce/config.yml
+          saucectl run
+        env:
+          SKIP_CI: true
```

Comment:
There shouldn't be any use for this any longer 💡

Reply:
will remove